### PR TITLE
Adds explosive collars

### PR DESCRIPTION
**Pull request by DiscordWizard**
## About The Pull Request

Adds a copy of a shock collar which is explosive instead, because I didn't want to copy so much code. It is certified to just end your life, unless you somehow find strong enough armor for your head and the rest of your body.

## Why It's Good For The Game

Well we didn't have explosive collars and I liked the idea of having something more _encouraging_ for my xenochimera character rather than a button that just makes them more prone to acting like an animal

Feel free to tell me to be less lazy with the code or remove it from the loadout section, I felt like it could be used for whatever people may need it, private or event wise

## Changelog

:cl:
add: Added explosive collars
/:cl:

### CODE REVIEW

**Review by silicons — CHANGES_REQUESTED** [review]
we're not nearly dystopian enough that i want literal bomb collars in the loadout

**Comment by Captain277** [comment]
> we're not nearly dystopian enough that i want literal bomb collars in the loadout

It's kinda hot though...

**Comment by DiscordWizard** [comment]
> we're not nearly dystopian enough that i want literal bomb collars in the loadout
I could put it on the xenowear section restricted to xenochimera if you want, It's the only character I wanted to use it on

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -770,6 +770,39 @@
...
+		if(ismob(loc))
+			M = loc
+		if(ismob(loc.loc))
+			M = loc.loc // This is about as terse as I can make my solution to the whole 'collar won't work when attached as accessory' thing.
+		to_chat(M,"<span class='danger'>You hear a soul wrenching beep!</span>")
+		var/turf/T = get_turf(M.loc)
+		if(T)
+			//Because I really want to see heads blow up
+			M.apply_damage(60, BURN, "head", used_weapon="Electrocution")
+			M.apply_damage(220, BRUTE, "head", used_weapon="Electrocution")
```
Comment (BlueWildrose) [inline code/modules/clothing/under/accessories/accessory.dm:800]:
```suggestion
			M.apply_damage(60, BURN, "head", used_weapon="Bomb Collar")
			M.apply_damage(220, BRUTE, "head", used_weapon="Bomb Collar")
```

**Review by BlueWildrose — CHANGES_REQUESTED** [review]
Pretty sure getting your head blown off wouldn't exactly be _electrocution_.

**Comment by silicons** [comment]
feel free to reopen if you want this as a feature without a loadout entry

**Pull request closed without merging**